runtime/astar/src/lib.rs

```diff
@@ -254,9 +269,9 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const BasicDeposit: Balance = 10 * ASTR;       // 258 bytes on-chain
-    pub const FieldDeposit: Balance = 25 * MILLIASTR;  // 66 bytes on-chain
-    pub const SubAccountDeposit: Balance = 2 * ASTR;   // 53 bytes on-chain
+    pub const BasicDeposit: Balance = deposit(1, 258);  // 258 bytes on-chain
+    pub const FieldDeposit: Balance = deposit(0, 66);  // 66 bytes on-chain
+    pub const SubAccountDeposit: Balance = deposit(1, 53);  // 53 bytes on-chain
```

Comment:
Is there a way to get actual sizeofs for on-chain bytes instead of hand written constants?

Reply:
Perhaps the size of the struct being stored could be fetched via `MaxEncodedLenTrait` but it's function isn't a const. So not right now, to the best of my knowledge - perhaps in the future.